client/src/components/game/lab4/pages/GameStart.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component, Fragment } from "react";
+import { navigate } from "@reach/router";
+import AppInstructions from "../components/AppInstructions";
+ import Button from "@material-ui/core/Button";
+
+class GameStart extends Component {
+    handleSubmit() {
+        navigate("/Lab4/Game/SmallTarget");
+    }
+
+    render() {
+        const instructions = "The next page will begin the game.";
+        const buttonStyle = { marginRight: "10px", marginLeft: "10px" };
```

Comment:
Maybe add marginBottom, the button is very close to the next/previous buttons

Reply:
good point! I added a top padding for the footer in general because I noticed it's super close to the content on every page

Reply:
Good catch!